lib/std/zig/parse.zig

```diff
@@ -2007,15 +2011,64 @@ const Parser = struct {
     /// Block <- LBRACE Statement* RBRACE
     fn parseBlock(p: *Parser) !Node.Index {
         const lbrace = p.eatToken(.l_brace) orelse return null_node;
+
         const scratch_top = p.scratch.items.len;
         defer p.scratch.shrinkRetainingCapacity(scratch_top);
+
+        const parent_block_tok = p.current_block_indent_defining_token;
+        defer p.current_block_indent_defining_token = parent_block_tok;
+        const parent_indent = p.tokenColumn(parent_block_tok);
```

Comment:
Minor performance note: This leads to some redundant scanning for the token column, but knowing the token (not just column) is useful for the error note. An alternative is to have `Parser` store a tuple of `token, column` like `var sibling` below.